docs/Speech and Audio/Setting Up Voice Commands/index.md

```diff
@@ -54,10 +54,15 @@ sdlManager.getScreenManager().setVoiceCommands([voiceCommand]);
 !@
 
 ### Unsupported Voice Commands
-The library automatically filters out empty strings and whitespace-only strings from a voice command's @![iOS, javascript]array!@@![android, javaSE, javaEE]list!@ of strings. For example, if a voice command has the following @![iOS, javascript]array!@@![android, javaSE, javaEE]list!@ values: `[" ", "First", "", "Voice Command"]` the library will filter it to: `["First", "Voice Command"]`.
+The library automatically filters out empty strings and whitespace-only strings from a voice command's @![iOS, javascript]array!@@![android, javaSE, javaEE]list!@ of strings. For example, if a voice command has the following @![iOS, javascript]array!@@![android, javaSE, javaEE]list!@ values: `[" ", "CommandA", "", "Command A"]` the library will filter it to: `["CommandA", "Command A"]`.
 
 If you provide @![iOS, javascript]an array!@@![android, javaSE, javaEE]a list!@ of voice commands which only contains empty string and whitespace-only strings across all of the voice commands, the upload request will be aborted and the previous voice commands will remain available.
 
+### Duplicate Strings in Voice Commands
+Voice commands that are sent with duplicate strings in different voice commands—such as: `(Command1: ["Command A", "Command B"], Command2: ["Command B", "Command C"], Command3: ["Command D", "Command E"])`—the manager will abort the upload request. The previous voice commands will remain available.
+
+If any individual voice command contains duplicate strings, they will be reduced to one. For example, if the voice commands to be sent are: `(Command1: ["Command A", "Command A", "Command B"], Command2: ["Command C", "Command D"])`, the manager will strip the duplicates to: `(Command1: ["Command A", "Command B"],  Command2: ["Command C", "Command D"])`.
```

Comment:
Change to:

If any individual voice command contains duplicate strings, they will be reduced to one. For example, if the voice commands to be sent are: 

```
{
    Command1: ["Command A", "Command A", "Command B"],
    Command2: ["Command C", "Command D"]
}
```

Then the manager will strip the duplicates to:

```
{
    Command1: ["Command A", "Command B"],
    Command2: ["Command C", "Command D"]
}
```